### ISSUE TITLE
An adjusted continuation is expressible but not labelled: kind, family and operation description lose the lineage-extension relation

### ISSUE THREAD

**Issue by Cissou34730** [Cissou34730/robot-learning-autoresearch#50]
## Problem

Continuing an existing lineage while making one adjustment is already expressible: `kind: "training"` with `initialization: "transfer"`, a `training_parent`, and `params` (and/or a researcher-owned code change). The capability is not missing.

What is missing is the **labelling and the record semantics**. The harness vocabulary reserves the word `continuation` for the strictly unchanged case, and every derived field - the operation description, the family, the experiment-index row, the kind table in the instructions - is computed from `kind`. So an adjusted continuation is recorded, grouped and displayed as an ordinary changed-method training run, and the fact that it extends lineage X's training is not expressed anywhere the Researcher reads as structure. The two forms of the same scientific operation also differ in which recipe is actually in effect, and the record does not make that difference legible either.

## Evidence

**`continuation` is defined as strictly unchanged** - `research/runner_protocol.py:608-609`:

```python
if experiment_kind == "continuation" and (parameter_overrides or code_changes):
    raise ValueError("continuation requires an unchanged learning method")
```

**The operation description is hard-coded from `kind`** - `research/runner_protocol.py:381-389`:

```python
def operation_description(record: dict) -> str:
    """Return the stable human-readable operation description for a record."""
    kind = str(record.get("kind", "")).strip().lower()
    if kind == "continuation":
        return "Continue training the unchanged method"
    if kind == "replication":
        return "Replicate the current method from fresh initialization"
    value = record.get("change")
    return value.strip() if isinstance(value, str) else ""
```

An adjusted continuation falls through to free-text `change`. Whether the record says it continues a lineage depends entirely on whether the Researcher happened to write that in prose.

**The family is derived so that it never expresses the lineage relation** - `research/runner_protocol.py:357-378`:

```python
parameter_paths = sorted({item["path"] for item in parameter_changes})
if parameter_paths:
    return "+".join(parameter_paths)
...
return experiment_kind
```

- An adjusted continuation gets the family of the touched parameter path, grouping it with *any* experiment touching that parameter, including fresh-initialization ones.
- An unchanged continuation gets the literal family `"continuation"`, grouping all continuations of all lineages together.

Neither family says which lineage was extended.

**The brief renders `kind / family` as the primary operation column** - `research/build_research_brief.py:908` and `research/build_research_brief.py:918`:

```python
"| # | Operation / family | Parent | Intervention | Measurements | Hypothesis assessment | Final decisions | Detail |",
...
f"| {result.get('index', '-')} | {result.get('kind', '-')} / {result.get('family', '-')} | "
```

So "extend lineage X further" appears as `continuation / continuation` in one row and as `training / learning.learning_rate` in another; and "continue lineage A" and "continue lineage B" appear as the same family. `_change_details` (`research/build_research_brief.py:159-171`) reinforces this: when `parameter_changes` exist it returns only the parameter delta and never calls `operation_description`.

**The two forms differ in which recipe is in effect, and only one is labelled** - `research/run_experiment.py:214-219`:

```python
parent = protocol.resolved_training_parent(proposal, state, initialization)
restore = (
    _serialize_restore_plan(protocol.plan_lineage_restore(parent))
    if parent is not None
    and str(proposal.get("kind", "training")).lower() == "continuation"
    else None
)
```

and `research/run_experiment.py:366-368`:

```python
parent = protocol.resolved_training_parent(proposal, raw_state, initialization)
if experiment_kind == "continuation" and parent is not None:
    protocol.plan_lineage_restore(parent)
```

A `continuation` restores the parent's complete recipe (its `scientific_commit` and `parameters`) before training. A `training` + `transfer` run trains the *current worktree science* plus the proposal's `params`, starting from the parent's weights. Both are legitimate, they are scientifically different, and nothing in the record's operation labelling distinguishes "extended P's recipe" from "applied today's recipe to P's weights".

**The instruction table presents `continuation` as the way to continue** - `research/instruments.md:256-257`:

```
| `training` | Trains a changed scientific recipe for any investigation type. | ... |
| `continuation` | Trains the unchanged method further from an eligible lineage. ... Code changes, parameter overrides and `change` are forbidden. |
```

The alternative appears only as prose far below the table, at `research/instruments.md:286-287`:

> `continuation` continues the selected recipe without a learning-method change. A `training` proposal may deliberately apply a changed recipe to an existing parent with `initialization: "transfer"`.

**A precedent for the correct shape already exists in the schema**: replication expresses its relation with a dedicated field rather than by overloading `kind` - `research/instruments.md:258` requires "a positive integer `replication_of` naming an existing experiment in the current campaign", and `research/instruments.md:299-301` states it "groups the new run with the referenced experiment for replication evidence". No equivalent relation field exists for lineage extension.

## Why this biases the decision

- **The kind table teaches a false dichotomy.** A Researcher reading `research/instruments.md:256-257` sees "continue = unchanged" and "training = changed", and resolves the tension by dropping one side: either it abandons the adjustment to keep the clean continuation label, or it abandons the lineage framing and reasons about the run as a fresh intervention. Both distort the hypothesis it then writes.
- **History reads wrong in exactly the place it is consulted.** The experiment index is the compact view the Researcher scans when deciding what has already been tried. In it, a campaign that in fact extended one lineage five times with small adjustments looks like five unrelated parameter experiments, while five unrelated continuations of different lineages look like one homogeneous block. Both errors point the next proposal at the wrong precedent.
- **The unchanged form is rhetorically privileged.** It is the only operation that comes with a stable, harness-authored description ("Continue training the unchanged method"). The adjusted form must be described by the Researcher in free text, and free text is truncated to 100 characters in the index (`research/build_research_brief.py:919`). Under any pressure toward legible records, the unchanged continuation is the safer-looking choice - which biases toward not adjusting.
- **The recipe-in-effect difference is invisible.** A Researcher choosing `training` + `transfer` to "continue P with a smaller learning rate" may not realise it is also silently adopting whatever the current worktree science is, rather than P's. The postmortem then attributes the outcome to the learning-rate change alone.

## Recommended remedy

Express the lineage relation as data, independent of `kind`, the way `replication_of` already does.

1. **Add an explicit relation field to the proposal and the experiment record**, e.g. `extends_lineage: true` (or `continues: "<lineage id or experiment index>"`), valid for any `training` proposal with `initialization: "transfer"` and a `training_parent`. Validate it in `research/runner_protocol.py` alongside the existing kind rules, and persist it into the experiment record next to `training_parent`.
2. **Make `operation_description` relation-aware** (`research/runner_protocol.py:381-389`): when the record extends a lineage, return a harness-authored description in the same register as the unchanged case, e.g. `f"Continue training lineage {parent} with a changed recipe"`, appending the Researcher's `change` text. Neither form should depend on free text for its basic character.
3. **Make `experiment_family` express the lineage** (`research/runner_protocol.py:357-378`): when a record extends a lineage, derive the family from the lineage rather than from the parameter path - for example `lineage.<parent>` or `lineage.<parent>+<parameter path>` - and stop returning the bare literal `"continuation"` for unchanged continuations so that continuations of different lineages are not pooled.
4. **Surface the relation in the brief**: in `research/build_research_brief.py:918-919`, render the operation column so a lineage extension is visibly one operation type regardless of whether it carried an adjustment, and change `_change_details` (`research/build_research_brief.py:159-171`) to prefix the parameter delta with the operation description instead of replacing it.
5. **Record the recipe basis explicitly.** Persist, for every transfer run, whether the parent's recipe was restored (`recipe_restore` is already computed at `research/run_experiment.py:215-219`) or whether the current worktree science was used, and render that distinction in the brief. Optionally - and this is the cleanest version - widen `kind: "continuation"` to accept `params` and keep `plan_lineage_restore` running for it, so "the parent's recipe plus this delta" and "today's recipe on the parent's weights" become two explicitly distinguishable, explicitly labelled operations. If that is done, `research/runner_protocol.py:608-609` must be relaxed and `research/runner_protocol.py:384-385` must stop asserting "unchanged".
6. **Rewrite the kind table** at `research/instruments.md:256-257` so the adjusted-continuation form appears *in the table*, with its required fields, rather than as prose at `research/instruments.md:286-287`. State plainly that continuing a lineage and changing the recipe are independent choices.
7. Add `tests/autoresearch/` coverage asserting that a transfer training run extending a lineage produces a family and an operation description that name the lineage, and that they are stable across records with and without a parameter delta.
